src/controllers/users.js

```diff
@@ -31,8 +31,10 @@ export default class UsersController {
       const response = user.toJSON();
 
       delete response.password;
-      const { id: userId } = user;
+      const { id: userId, firstName } = user;
       const token = await generateToken({ userId });
+      const url = `${req.headers.host}/api/v1/users/confirmAccount?token=${token}`;
+      // await sendEmail(email, 'confirmAccount', { firstName, url });
```

Comment:
If this line is commented out, how is the email being sent?

Reply:
Because the email as not been merged. I branched off  `develop` so that I can implement it. Once the email is merged, I'll pull and uncomment this line. You can test by sending the url in the as part of the response object

Reply:
Alright. I just wanted to understand that part. Kudos.